### ISSUE TITLE
LSP as a standalone instalation

### ISSUE THREAD

**Issue by yassinebridi** [zenstackhq/zenstack#1156]
**Is your feature request related to a problem? Please describe.**
I noticed in the documentation that there is plugins for VS Code and JetBrains, wouldn't be better to provide the LSP as a standalone installation like prisma [does](https://www.npmjs.com/package/@prisma/language-server), and everyone can install the LSP in their favorite editor.

**Describe the solution you'd like**
A standalone LSP.

**Comment by ymc9** [comment]
Yes, right now the LSP resides in the same package as the CLI and VSCode extension, which doesn't make much sense. I hope we can address it in V2.

Btw, what IDE are you trying to use it with?

**Comment by yassinebridi** [comment]
I'm trying to use it with Neovim specifically:
https://github.com/neovim/nvim-lspconfig

**Comment by florescercadojulio** [comment]
Team of zed editor users here tasked with developing an application from scratch. We were looking at zenstack because it looks really nice when it comes to permissions and we decided to use it, BUT, having no LSP to place into our editor was a huge turn off. Having this would open up to a huge market. Would love to see it in the near future.
